.gitignore

```diff
@@ -207,6 +207,7 @@ FakesAssemblies/
 
 # Project
 TaskRouter.Web/Local.config
+TaskRouter.Web/Scripts/lib/
```

Comment:
Visual Studio will complain if there is no `lib` folder already present, so I suggest creating the folder and commiting to the repo with a `.gitkeep` file.

Here the entry can be like this:
```
# Libman
**/Scripts/lib/
!.gitkeep
```

Reply:
Actually it does not complain. Just tried removing the directory and rebuilding the project and it brings the folder back again with all the libraries.